app/_src/gateway/kong-manager/auth/oidc/migrate.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Migrate from Previous Configurations
+badge: enterprise
+---
+
+# Breaking changes
+
+## admin_gui_auth_conf
+
+### scopes
+
+While using the OpenID Connect plugin with Kong Manager, `scopes` now have a default value of
+`["openid", "email", "offline_access"]` if not specified.
+
+* `openid`: Essential for OpenID Connect
+* `email`: To retrieve user's email address and include it in the ID token
+* `offline_access`: Essential refresh tokens to refresh the access tokens and sessions
+
+This parameter can be modified according to needs. However, `"openid"` and `"offline_access"` should
+always be included to ensure the OpenID Connect plugin works normally. Also, making sure that `scopes`
+contains sufficient scopes for the claim specified by this parameter.
+
+### admin_claim
+
+`admin_claim` is now an optional parameter which has a default value of `["email"]` if not specified.
+
+This parameter is used while looking up for admin's username from the ID token. While configuring this,
+making sure that `scopes` contains sufficient scopes for the claim specified by this parameter.
+
+### authenticated_groups_claim
+
+`authenticated_groups_claim` is now an optional parameter which has a default value of `["groups"]`
+if not specified.
+
+This parameter is used while looking up for admin's associated groups from the ID token.
+
+### redirect_uri
+
+`redirect_uri` now should be configured to an array of URLs that points to Admin API's authentication
+endpoint (e.g., `["http://localhost:8001/auth"]`). Whereas, `redirect_uri` were previously URLs
+pointing to Kong Manager (e.g., `["http://localhost:8002"]`).
+
+### login_redirect_uri
+
+`login_redirect_uri` now becomes a **required** parameter to configure the destination after authenticating
+with the IdP. It should be always be an array of URLs that points to the Kong Manager
+(e.g., `["http://localhost:8002"]`).
+
+### logout_redirect_uri
+
+`logout_redirect_uri` now becomes a **required** parameter to configure the destination after logging
+out from the IdP. It should be always be an array of URLs that points to the Kong Manager
+(e.g., `["http://localhost:8002"]`).
+
+Previously, Kong Manager will not perform an [RP-initiated logout](https://openid.net/specs/openid-connect-rpinitiated-1_0.html#RPLogout)
+from the IdP when a user request to logout. From Gateway v3.6 and onwards, Kong Manager will perform
+an RP-initiated logout upon user logout.
+
+## admin_gui_session_conf
```

Comment:
What will happen if we don't update it?

Reply:
@raoxiaoyan If a user does not have any parameters configured for session management, the default session configuration provided by the OIDC plugin (which is slightly different from the one provided by the session plugin) will be used. 

For instance, a user may find the default value changed if they neither have `admin_gui_session_conf` nor session-related parameters configured in `admin_gui_auth_conf`.

I will add an extra warning here to ensure that the user won't miss this out.

Reply:
Sounds good. It looks like doesn't affect the customer's use.